### PR TITLE
Release 0.6.2

### PR DESCRIPTION
**Pull request by rfairley**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+### v0.6.2
+
+Locksmith now ignores D-Bus signals that are `nil` or have fields of incorrect
+type, when waiting for a reboot signal from update_engine. This avoids a `panic`
```
Comment (lucab) [inline CHANGELOG.md]: This should be something that makes sense to end-users who are reading this without prior knowledge about locksmith internals or code structure. I would stick to something simpler like:
```
Locksmith now internally handles DBus signals in a safer way, with additional checks on unexpected/malformed ones. This avoids a panic in the daemon component, which was frequently occurring on shutdown.
```

(I'm not a native speaker, so feel free to reword as you prefer)

Comment (rfairley) [reply CHANGELOG.md]: Makes sense, thanks! Will make this revision.

Comment (rfairley) [reply CHANGELOG.md]: Change committed - wording conveys what is needed.

**Review by ajeddeloh — APPROVED** [review]

**Review by lucab — APPROVED** [review]

**Comment by lucab** [comment]
I merged this, tagged 0.6.2 and forwarded it to coreos-overlay as the next alpha freeze is today and I am in a favourable timezone.

**Comment by rfairley** [comment]
Thanks, I see the PR in coreos-overlay. Will remember to tag the release and forward to coreos-overlay next time to complete a fix.

**Pull request merged**